tensorrt_llm/_torch/auto_deploy/models/phi.py

```diff
@@ -137,17 +137,17 @@ def _patch_phi3_emb_with_decorator_forward(self, x, position_ids):
     return cos.to(dtype=x.dtype), sin.to(dtype=x.dtype)
 
 
-def get_model_from_config_patched(model_config, trust_remote_code):
-    model = _from_config_original(model_config, trust_remote_code=trust_remote_code)
-    if re.search(r"Phi-4-mini-instruct", model_config._name_or_path):
+def get_model_from_config_patched(config, **kwargs):
+    model = _from_config_original(config, **kwargs)
+    if re.search(r"Phi-4-mini-instruct", config._name_or_path):
```

Comment:
```suggestion
    if re.search(r"Phi-4-mini-instruct", config.get(_name_or_path, "")):
```

Let's make it extra fault tolerant since we are already on it